lib/Conversion/TorchToLinalg/Pooling.cpp

```diff
@@ -72,10 +117,72 @@ checkAndGetPoolingParameters(OpTy op, ConversionPatternRewriter &rewriter,
   return success();
 }
 
+
+// Creates a pooling operation based on the type specified by `OpTy` and
+// arguments passed.
+template <typename OpTy>
+static LogicalResult createPooling1dOp(
```

Comment:
This lowering is very similar to the 2d case. The common logic should be identified and refactored to avoid repetition.

Reply:
> This lowering is very similar to the 2d case. The common logic should be identified and refactored to avoid repetition.

Same, Fixed by Merge them together